api/uptime-crawler.js

```diff
@@ -0,0 +1,92 @@
+import * as solanaWeb3 from '@solana/web3.js';
+import _ from 'lodash';
+import YAML from 'yaml';
+import redis from 'redis';
+import {promisify} from 'util';
+import {exec} from 'child_process';
+
+import config from './config';
+
+const SOLANA_WALLET_PATH =
+  process.env.SOLANA_WALLET_PATH || '../solana/target/debug';
```

Comment:
Do you know of a good npm module to find a file in PATH?   It would be nice to first search PATH for `solana-wallet`

Reply:
Good idea, will investigate...

Reply:
(implemented)